app/Actions/Place/AddPlaceValidator.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Hedonist\Actions\Place;
+
+
+use Illuminate\Support\Facades\Validator;
+
+class AddPlaceValidator
```

Comment:
You can create class and extend it from HttpRequest, move your validation there and then put data from that request into DTO request of Action class